### DISCUSSION TITLE
Add the option for rewards every X vote

### DISCUSSION THREAD

**Discussion by Recliness** [Zrips/CMI#8128] [Feature Requests]
This would be convenient so I don't have to copy-paste hundreds of lines in the vote rewards config. 

For example, every 10 votes a player would get a crate key. Rather than me having to put every 10th number to 1000 in the config, it would be nice if I could just put something like 

CommandInterval: 10

**Comment by CloudeLecaw** [comment]
#8132
 There i requested the useage of specialized commands in vote rewards.

when this would be allowed for use we also could make thinks like each 10th vote with cmis usermeta checks.
so i pointed it here in your request too.
